### PR TITLE
almost perfect

### PR DESCRIPTION
**Pull request by ctech4285**
Hmm I need some clarification on proper variable use; unsigned vout =  board::getOutVoltageInVolts(); 
a few empty lines need to be removed

### CODE REVIEW

**Comment by ctech4285** [comment]
Oh yeah, also not sure if I did the averaging right in board.cpp

File: firmware/src/magnet/charger.cpp [hunk 1]
```diff
@@ -77,6 +79,17 @@ Charger::Status Charger::runAndGetStatus()
 #if BOARD_OLIMEX_LPC_P11C24
         return Status::Done;                            // This is just a testing mock
 #endif
+        unsigned vout =  board::getOutVoltageInVolts();
```
Comment (ctech4285) [inline firmware/src/magnet/charger.cpp]: should the variable be static?

File: firmware/src/sys/board.cpp [hunk 2]
```diff
@@ -497,17 +497,16 @@ unsigned getSupplyVoltageInMillivolts()     //error under 2%
...
+    std::uint16_t value1 = 0;
+    (void)Chip_ADC_ReadValue(LPC_ADC, ADC_CH0, &value1);
 
+    std::uint16_t value2 = 0;
+    (void)Chip_ADC_ReadValue(LPC_ADC, ADC_CH0, &value2);
     // Division and multiplication by 2 are reduced
     const unsigned x =
-        static_cast<unsigned>((static_cast<unsigned>(new_value + old_value) * AdcReferenceMillivolts) >>
+        static_cast<unsigned>((static_cast<unsigned>(value1 + value2) * AdcReferenceMillivolts) >>
                               AdcResolutionBits);
```
Comment (ctech4285) [inline firmware/src/sys/board.cpp]: This actually doesn't work?

File: firmware/src/magnet/charger.cpp (hunk 1 above)
Comment (pavel-kirienko) [inline firmware/src/magnet/charger.cpp]: No, it has to be `const`.

File: firmware/src/sys/board.cpp (hunk 2 above)
Comment (pavel-kirienko) [inline firmware/src/sys/board.cpp]: This looks correct. Why doesn't it work?

Comment (ctech4285) [inline firmware/src/sys/board.cpp]: It returns half the real value

File: firmware/src/magnet/charger.cpp [hunk 3]
```diff
@@ -136,7 +157,10 @@ Charger::Status Charger::runAndGetStatus()
     {
         board::runPump(50, on_time_cy, off_time_cy);
     }
-
+    if (board::getOutVoltageInVolts() >= target_output_voltage_)
+    {
+         board::syslog(" Vin min = ", supply_volatage_mV_min , " mV\r\n");
```
Comment (ctech4285) [inline firmware/src/magnet/charger.cpp]: This give a compiler error, variable not declared in scope, what am I doing wrong?

File: firmware/src/sys/board.cpp [hunk 4]
```diff
@@ -497,18 +497,14 @@ unsigned getSupplyVoltageInMillivolts()     //error under 2%
 
 unsigned getOutVoltageInVolts()
 {
-    static std::uint16_t old_value = 0;
-
-    std::uint16_t new_value = 0;
-    (void)Chip_ADC_ReadValue(LPC_ADC, ADC_CH0, &new_value);
-
+    std::uint16_t result = 0;
```
Comment (ctech4285) [inline firmware/src/sys/board.cpp]: Hmm...I tried a bunch of things but could not get averaging to work right,...so now it's gone. Tested on a few units, noise seems to be very low

**Comment by ctech4285** [comment]
Everything tested, working and pretty, merging

**Pull request merged**